src/types/sections.d.ts

```diff
@@ -120,3 +120,59 @@ export interface AccordionSectionProps {
   items: Pick<AccordionProps, "title" | "text">[];
 }
 export class AccordionSection extends Component<AccordionSectionProps> {}
+
+export interface GoogleMapsSectionProps {
+  /**
+   * Your GoogleMaps apikey
+   */
+  apikey: string;
+  /**
+   * The location
```

Comment:
We should mention that these properties (startLocation, title, zoom, ...) are optional and not required.